test/document_stores/test_weaviate.py

```diff
@@ -144,3 +144,27 @@ def test_deleting_by_id_or_by_filters(document_store_with_docs):
     document_store_with_docs.delete_documents(filters={"name": ["filename2"]})
     document_store_with_docs.get_all_documents.assert_called()
     assert document_store_with_docs.get_document_count() == 1
+
+
+def get_document_store_with_docs_and_similarity(tmp_path, similarity, recreate_index):
+    document_store = get_document_store(
+        "weaviate", tmp_path=tmp_path, similarity=similarity, recreate_index=recreate_index
+    )
+    document_store.write_documents(DOCUMENTS_XS)
+    yield document_store
+    document_store.delete_index(document_store.index)
```

Comment:
As the tests got simplified, this is not needed anymore.

Reply:
You are right, I forgot to remove that.